src/create-config.ts

```diff
@@ -37,6 +37,18 @@ const questions = (choices: Choice[]): PromptObject[] => {
   ];
 };
 
+export const getStageTimestamp = () => {
+  const now = new Date();
+  const year = now.toLocaleString('en-US', { year: '2-digit' });
+  const month = now.toLocaleString('en-US', { month: '2-digit' });
+  const day = now.toLocaleString('en-US', { day: '2-digit' });
+  const hour = now.toLocaleString('en-US', { hour: '2-digit', hour12: false });
+  // const minute = now.toLocaleString('en-US', { minute: '2-digit' }); // doesn't work
```

Comment:
This works for me, but I'm not fussed, just interesting. This is in `Node.js v14.17.1`.

Reply:
Thanks @aquarat ! 

The issue that I'm facing with the commented out code is that it doesn't honor the `2-digit` when the number of minutes is less that 10:
![image](https://user-images.githubusercontent.com/5776880/174302622-80a654c6-7149-4e8d-9aee-6eefd880b60b.png)
This seems to only be an issue with minutes tho.

As an alternative I could have just used numbers and prepend '0' for single digits:
```
export const getStageTimestamp = () => {
  const now = new Date();
  const year = now.getFullYear() % 100;
  const month = ('0' + (now.getMonth() + 1)).slice(-2);
  const day = ('0' + now.getDay()).slice(-2);
  const hour = ('0' + now.getHours()).slice(-2);
  const minute = ('0' + now.getMinutes()).slice(-2);

  return `${year}${month}${day}-${hour}${minute}`;
};
```
but I'm not convinced it is less ugly that my first approach 🤭

Reply:
Interesting, thanks, again I'm not fussed, just curious :)